sequencer/src/main.rs

```diff
@@ -18,10 +18,10 @@ async fn main() -> anyhow::Result<()> {
     setup_logging();
     setup_backtrace();
 
-    tracing::warn!("sequencer starting up");
```

Comment:
@jbearer just saw you recently changed this. Feels sort of wrong to log with warn or error level if there's no cause for alarm. Wondering why this was changed.